src/axom/primal/geometry/Vector.hpp

```diff
@@ -537,6 +592,13 @@ AXOM_HOST_DEVICE inline T Vector<T, NDIMS>::dot(const Vector<T, NDIMS>& vec) con
   return dot_product(*this, vec);
 }
 
+//------------------------------------------------------------------------------
+template <typename T, int NDIMS>
+AXOM_HOST_DEVICE inline T Vector<T, NDIMS>::cross(const Vector<T, NDIMS>& vec) const
+{
+  return cross_product(*this, vec);
+}
```

Comment:
We might need to be careful about this one -- the current `cross_product` overloads are defined as:
* `cross_product(Vector3, Vector3) -> Vector3`
* `cross_product(Vector2, Vector2) -> Vector3`  // where the x- and y- components are 0

https://github.com/LLNL/axom/blob/379db95ceed3ce8e90145993a0d9c33f8103f330/src/axom/primal/geometry/Vector.hpp#L567-L586